libseccomp/src/filter_context.rs

```diff
@@ -538,6 +629,21 @@ impl ScmpFilterContext {
     ///
     /// If this function is called with an invalid filter, an issue is encountered
     /// getting the current state, or the libseccomp API level is less than 4, an error will be returned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use libseccomp::*;
+    /// let mut ctx = ScmpFilterContext::new_filter(ScmpAction::Allow)?;
+    /// if check_api(4, ScmpVersion::from((2, 5, 0))).unwrap() {
+    ///     ctx.set_ctl_ssb(false)?;
+    ///     assert!(!ctx.get_ctl_ssb()?);
+    /// } else {
+    ///     assert!(ctx.set_ctl_ssb(false).is_err());
+    ///     assert!(ctx.get_ctl_ssb().is_err());
+    /// }
```

Comment:
This else is not necessary in example code.

Reply:
My idea (about negative out come because of version/api-level) for this case as an exmaple:
```rust
    /// # use libseccomp::*;
    /// let mut ctx = ScmpFilterContext::new_filter(ScmpAction::Allow)?;
    /// # if check_api(4, ScmpVersion::from((2, 5, 0))).unwrap() {
    /// ctx.set_ctl_ssb(false)?;
    /// assert!(!ctx.get_ctl_ssb()?);
    /// # }
    /// # Ok::<(), Box<dyn std::error::Error>>(())
```